test/Engine/ProtoTest/Associative/MethodsFocusTeam.cs

```diff
@@ -1600,29 +1592,29 @@ public void Test()
         }
 
         [Test]
-        [Category("DSDefinedClass")]
+        [Category("DSDefinedClass_Ported")]
         public void T056_nonmatchingclass_1467162()
         {
             String code =
-            @"
                class A
                {
                    fa = 1;
                }
                a:M = A.A();//
            ";
+            @"
                import(""FFITarget.dll"");
                a : M = DummyPoint.ByCoordinates(10, 20, 30);
            ";
             thisTest.RunScriptSource(code);
             thisTest.Verify("a", null);
             TestFrameWork.VerifyRuntimeWarning(ProtoCore.Runtime.WarningID.kConversionNotPossible);
         }
 
         [Test]
-        [Category("DSDefinedClass")]
+        [Category("DSDefinedClass_Ported")]
         public void T057_nonmatchingclass_1467162_2()

```

Comment:
The test was actually passing. But Ive restored it the [failure] since its a DSDefinedClass test that cannot be ported. It will be removed anyways